### PR TITLE
Accept request content as "data"

### PR DESCRIPTION
**Pull request by jviotti**
For legacy support.

### CODE REVIEW

**Comment by jviotti** [comment]
Ignore Hound CI comments. Configured correctly in another PR.

**Comment by emirotin** [comment]
lgtm

File: lib/pine.coffee [hunk 1]
```diff
@@ -11,6 +11,10 @@ class ResinPine extends PinejsClientCore
 		params.json = true
 		params.gzip ?= true
 
+		# Support "data" attribute
+		if params.data? and not params.body?
```
Comment (Page-) [inline lib/pine.coffee:15]: `data` is never passed to `_request` by pinejs-client, so this code will never be triggered in normal use (only if someone explicitly calls `_request` which is unsupported)

**Comment by Page-** [comment]
No released version of pinejs-client uses `data`, or even passes it to `_request` if someone mistakenly uses it - so I'd say NAK

File: lib/pine.coffee (hunk 1 above)
Comment (jviotti) [inline lib/pine.coffee:15]: True, I tested this branch on the SDK and `data` is not even passed, therefore this PR is irrelevant. I'll close for now.

**Pull request closed without merging**